app/Http/Regions/ForumRow.php

```diff
@@ -8,8 +8,10 @@ public function render($forum, $route = '')
     {
         $user = request()->user();
         $commentCount = $forum->vars()->commentCount;
-        $unreadCommentCount = $forum->vars()->unreadCommentCount;
-        $firstUnreadCommentId = $forum->vars()->firstUnreadCommentId;
+        //$unreadCommentCount = $forum->vars()->unreadCommentCount;
+        $unreadCommentCount = $forum->unread;
```

Comment:
Lets keep `$forum->vars()->unreadCommentCount` as a standard naming convention. You could do it in followup of course.